pandas/tests/indexing/interval/test_interval.py

```diff
@@ -114,6 +114,10 @@ def test_loc_getitem_frame(self):
         with pytest.raises(KeyError, match="^$"):
             df.loc[[10, 4]]
 
+    def test_print_intervalindex(self):
```

Comment:
is the issue actualy with str? if so test that if its *actually* with print, then do this
```
def test_print_interval_index(capsys):
    print(....)
    captured = capsys.readouterr()
    result = captured.out
    # assert
```

Reply:
I was unable to reproduce the OP (even with the environment details posted in the issue) but looking at the traceback I think the issue was with the string representation